spec/classes/backend_spec.rb

```diff
@@ -21,6 +21,14 @@
         it { should contain_class('sensu::backend::default_resources') }
         it { should_not contain_class('sensu::backend::datastore::postgresql') }
 
+        context 'on systemd host', if: Puppet.version.to_s =~ %r{^5} do
```

Comment:
shouldn't this also match 6.0

Reply:
No, because we don't rspec unit test against older version of Puppet 5. We unit test against latest 5.x and latest 6.x.